_rules/auto-update-text-efbfc7.md

```diff
@@ -13,32 +13,32 @@ accessibility_requirements: # Remove whatever is not applicable
 input_aspects:
   - DOM Tree
   - CSS Styling
-acknowledgements:
+acknowledgments:
   authors:
     - Carlos Duarte
 ---
 
 ## Applicability
 
-The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
+The rule applies to any [visible][] [HTML element][] if:
 
-- **(not alone)** it is not the only [content][] in the [HTML document][]; and
 - **(changed)** the `innerText` property of the [element][html element] changes; and
 - **(no child changed)** the [element][html element] does not have [children][child] whose `innerText` property also changes; and
+- **(not alone)** the [element][html element] has [sibling][] or [ancestor][] elements with non-empty `innerText` properties; and
```

Comment:
I guess for ancestors we also need the non-empty `innerText` to be different from the `innerText` of the changing element.